src/components/Nav/Nav-main.scss

```
@@ -0,0 +1,46 @@
+@import '../../styles/variables';
+.nav-main {
+  position: relative;
+  .nav-contaner-scrolled {
+    display: block;
```

```suggestion

.nav-main {
  position: relative;

    .nav-contaner-scrolled {
    display: block;
```

가독성을 위해서 한칸씩 띄워서 작성해주세요!